lib/hacluster.pm

```diff
@@ -60,6 +60,7 @@ our @EXPORT = qw(
   rsc_cleanup
   ha_export_logs
   check_cluster_state
+  check_online_nodes
```

Comment:
Does really need to be exported? Is there any other code place out of this file using it?

Reply:
No other place, but function seemed general enough that I think can be called outside of the library too.

Reply:
I'll add unit tests for this one too.

Reply:
After discussing with @mpagot in slack, we decided not to export the function. POD has been updated to reflect this. Whenever we decide to use this function outside of the library, it will need to be exported.